src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,73 @@
+type PaginationType = {
+  total?: number;
+  perPage?: number;
+  currentPage?: number;
+  pages: string[][];
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination = ({
+  currentPage = 1,
+  pages,
```

Comment:
Remove the `pages` prop from the component's props. Use `total` and `perPage` to determine the number of pages.